src/Animations/Animations/Animations.csproj

```diff
@@ -1,13 +1,72 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <AssemblyName>XamarinCommunityToolkit.Animations</AssemblyName>
-    <RootNamespace>XamarinCommunityToolkit.Animations</RootNamespace>
+    <TargetFrameworks>netstandard1.0;netstandard2.0;Xamarin.iOS10;MonoAndroid80;uap10.0.16299</TargetFrameworks>
+    <AssemblyName>Xamarin.Toolkit.Animations</AssemblyName>
+    <RootNamespace>Xamarin.Toolkit.Animations</RootNamespace>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <Product>$(AssemblyName) ($(TargetFramework))</Product>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <AssemblyFileVersion>1.0.0.0</AssemblyFileVersion>
+    <Version>1.0.0.0</Version>
+    <PackageVersion>1.0.0.0</PackageVersion>
+    <Authors>Xamarin Inc.</Authors>
```

Comment:
Should this be Microsoft?

Reply:
I just followed the Xamarin.Forms nuget...?